app/assets/javascripts/app/views/contacts_searcher_view.js

```diff
@@ -0,0 +1,122 @@
+app.views.ContactsSearcher = app.views.Base.extend({
```

Comment:
The name is a bit weird tbh. How about `ContactAutocomplete` or `ContactSearch`?